probePlugin/src/main/scala/org/virtuslab/ideprobe/handlers/RunConfigurations.scala

```diff
@@ -1,35 +1,39 @@
 package org.virtuslab.ideprobe.handlers
 
-import java.util.Collections
-import java.util.UUID
 import java.util.concurrent.CountDownLatch
+import java.util.{Collections, UUID}
+
 import com.intellij.compiler.options.CompileStepBeforeRun.MakeBeforeRunTask
-import com.intellij.execution.ExecutionManager
-import com.intellij.execution.RunManager
-import com.intellij.execution.RunnerAndConfigurationSettings
+import com.intellij.execution._
 import com.intellij.execution.application.ApplicationConfiguration
 import com.intellij.execution.executors.DefaultRunExecutor
-import com.intellij.execution.impl.RunManagerImpl
-import com.intellij.execution.impl.RunnerAndConfigurationSettingsImpl
+import com.intellij.execution.impl.{RunManagerImpl, RunnerAndConfigurationSettingsImpl}
 import com.intellij.execution.junit.JUnitConfiguration
 import com.intellij.execution.runners.ExecutionUtil
-import com.intellij.execution.testframework.sm.runner.SMTRunnerEventsAdapter
-import com.intellij.execution.testframework.sm.runner.SMTRunnerEventsListener
-import com.intellij.execution.testframework.sm.runner.SMTestProxy
-import com.intellij.openapi.project.DumbService
-import com.intellij.openapi.project.Project
+import com.intellij.execution.testframework.sm.runner.{SMTRunnerEventsAdapter, SMTRunnerEventsListener, SMTestProxy}
+import com.intellij.openapi.project.{DumbService, Project}
 import com.intellij.psi.JavaPsiFacade
 import com.intellij.psi.search.GlobalSearchScope
+import org.jetbrains.plugins.scala.testingSupport.test.scalatest.{
+  ScalaTestConfigurationType,
+  ScalaTestRunConfigurationFactory
+}
+import org.jetbrains.plugins.scala.testingSupport.test.testdata.{AllInPackageTestData, ClassTestData, SingleTestData}
+import org.jetbrains.plugins.scala.testingSupport.test.{AbstractTestRunConfiguration, TestKind}
+import org.virtuslab.ideprobe.Extensions._
 import org.virtuslab.ideprobe.RunnerSettingsWithProcessOutput
-import org.virtuslab.ideprobe.protocol.ApplicationRunConfiguration
-import org.virtuslab.ideprobe.protocol.JUnitRunConfiguration
-import org.virtuslab.ideprobe.protocol.ProcessResult
-import org.virtuslab.ideprobe.protocol.TestRun
-import org.virtuslab.ideprobe.protocol.TestStatus
-import org.virtuslab.ideprobe.protocol.TestSuite
-import org.virtuslab.ideprobe.protocol.TestsRunResult
+import org.virtuslab.ideprobe.protocol.{
+  ApplicationRunConfiguration,
+  JUnitRunConfiguration,
+  ProcessResult,
+  TestRun,
+  TestStatus,
+  TestSuite,
+  TestsRunResult,
+  _
```

Comment:
Importing a bunch of single classes is not necessary when you also import the entire package with a wildcard.